proto/self_serve/orb/orb.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package self_serve.orb;
+
+enum OrbMode {
+  ORB_MODE_UNSPECIFIED = 0;
+  LEGACY = 1;
+  SELF_SERVE = 2;
+}
+
+enum OrbType {
+  ORB_TYPE_UNSPECIFIED = 0;
+  PEARL = 1;
+  DIAMOND = 2;
+}
+
+message OrbMetadata {
```

Comment:
This is a replacement for `AnnounceOrbId` message.
This message is not versioned because we need the `orb_mode` field to conclude which version of messaging should we use.

This message should not be changed, we can only add fields to it.